opensearch/opensearch-kafka/readme.md

```diff
@@ -0,0 +1,55 @@
+# OpenSearch Example
+
+This repo is intended to provide the SA team with a means to populate an OpenSearch index with data so that you can demonstrate how OpenSearch can be used.
```

Comment:
Perhaps change this description since it's opened up to the public?

Reply:
changed, thank you